lib/collection/src/collection_manager/holders/proxy_segment.rs

```diff
@@ -463,7 +478,13 @@ impl SegmentEntry for ProxySegment {
                     was_deleted = self
                         .deleted_points
                         .write()
-                        .insert(point_id, op_num)
+                        .insert(
+                            point_id,
+                            ProxyDeletedPoint {
+                                local_version: op_num,
+                                operation_version: op_num,
+                            },
```

Comment:
For explicit delete operations I use `op_num` for both versions. It prevents moving the point into the write segment through another proxy, if it has a point version in between this local one and this operation.